ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource

```diff
@@ -234,77 +235,76 @@ Open Dashboard Settings
-    ${is_checked}=  Run Keyword And Return Status  Checkbox Should Be Selected  ${SINGLE_MODEL_SERVING_CHECK_BOX}
+    ${is_checked}=  Run Keyword And Return Status  Checkbox Should Be Selected  ${SINGLE_MODE_SERVING_CHECK_BOX}
     RETURN  ${is_checked}
 
-Click CheckBox Single Model Serving Platforms
+Select CheckBox Single Model Serving Platforms
     [Documentation]    Enables Single model serving platform
-    ${status}=  Get Checkbox State Of Single Modelserving platforms
-    IF  "${status}"=="False"
-        Select Checkbox   ${SINGLE_MODEL_SERVING_CHECK_BOX}
-    END
+    Select Checkbox   ${SINGLE_MODE_SERVING_CHECK_BOX}
+
 
-Click CheckBox Multi Model Serving Platforms
+Select CheckBox Multi Model Serving Platforms
     [Documentation]    Enables Multi Modelserving platforms
-    ${status}=  Get Checkbox State Of Multi Modelserving platforms
-    IF  "${status}"=="False"
-        Select Checkbox   ${MULTI_MODEL_SERVING_CHECK_BOX}
-    END
+    Select Checkbox   ${MULTI_MODE_SERVING_CHECK_BOX}
 
-UnClick CheckBox Single Model Serving Platforms
+Deselect CheckBox Single Model Serving Platforms
     [Documentation]    Disable checkbox Single Modelserving platforms
-    ${status}=  Get Checkbox State Of Single Modelserving platforms
-    IF  "${status}"=="True"
-        Unselect Checkbox   ${SINGLE_MODEL_SERVING_CHECK_BOX}
-    END
+    Unselect Checkbox   ${SINGLE_MODE_SERVING_CHECK_BOX}
 
-UnClick CheckBox Multi Model Serving Platforms
+Deselect CheckBox Multi Model Serving Platforms
     [Documentation]    Disable checkbox Multi Modelserving platforms
-    ${status}=  Get Checkbox State Of Single Modelserving platforms
-    IF  "${status}"=="True"
-        Unselect Checkbox   ${MULTI_MODEL_SERVING_CHECK_BOX}
-    END
+    Unselect Checkbox   ${MULTI_MODE_SERVING_CHECK_BOX}
 
-Click Both Model Serving Platforms
+Select Both Model Serving Platforms
     [Documentation]    Enables Multi Modeland Single Modelserving platforms
-    Click CheckBox Single Model Serving Platforms
-    Click CheckBox Multi Model Serving Platforms
+    Select CheckBox Single Model Serving Platforms
+    Select CheckBox Multi Model Serving Platforms
 
-UnClick Both Model Serving Platforms
+Deselect Both Model Serving Platforms
     [Documentation]    Disbale Multi Modeland Single Modelserving platforms
-    UnClick CheckBox Single Model Serving Platforms
-    UnClick CheckBox Multi Model Serving Platforms
+    Deselect CheckBox Single Model Serving Platforms
+    Deselect CheckBox Multi Model Serving Platforms
+
 
 Check IF Single Model Serving Is Disabled From DSC And Cluster Setting
     [Documentation]    Check single model should be disabled from DSC and dashboard
     Component Should Not Be Enabled  kserve
     ${status}=   Get Checkbox State Of Single Model Serving Platforms
-    Should Be Equal As Strings    ${status}  False
-    Element Should Be Disabled    ${SINGLE_MODEL_SERVING_CHECK_BOX}
+    ${false_status}=  BuiltIn.Convert To Boolean   False
```

Comment:
are you sure this is needed? Besides you're comparining as Strings in the next line, so this shouldn't be needed

Reply:
you can just do `Should Be Equal ${status} ${FALSE}`